.vscode/launch.json

```diff
@@ -19,6 +19,18 @@
         "script": "watch"
       }
     },
+    {
+      "type": "extensionHost",
+      "request": "launch",
+      "name": "Launch Prod Client",
+      "runtimeExecutable": "${execPath}",
+      "args": [
+        "--disable-extensions",
```

Comment:
May I suggest maybe removing the `--disable-extensions` from either the dev client or the prod client? Or maybe adding yet another config to include extensions for debugging g3?

Reply:
Yeah, I think it makes sense to not disable extensions when testing the prod build, since we'll most likely use this config to test g3.